ruby/tests/basic.rb

```diff
@@ -17,7 +17,6 @@ module BasicTest
     add_message "BadFieldNames" do
       optional :dup, :int32, 1
       optional :class, :int32, 2
-      optional :"a.b", :int32, 3
```

Comment:
Why this is not needed?

Reply:
It's an illegal field name. This is checked more robustly now, so the `add_message` call itself will fail now.